src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -12,8 +12,8 @@
 namespace Symfony\UX\TwigComponent;
 
 use Symfony\UX\StimulusBundle\Dto\StimulusAttributes;
-use Symfony\UX\TwigComponent\Escaper\HtmlAttributeEscaperInterface;
 use Symfony\WebpackEncoreBundle\Dto\AbstractStimulusDto;
+use Twig\Runtime\EscaperRuntime;
 
 /**
```

Comment:
Since we can do some BCs due to the security fixes, maybe... we can make this class internal?

Reply:
ComponentAttributes ? I agree. If we can have a confirmation ? cc @nicolas-grekas @fabpot  ?

Reply:
cross-component class, likely not a good idea

Reply:
Ok !